src/development/vortex/development/utils/data/collater/detr.py

```diff
@@ -0,0 +1,46 @@
+import torch
+
+from vortex.development.networks.models.detection.detr import NestedTensor
+from easydict import EasyDict
+
+class DETRColatte:
```

Comment:
should be DETRCollate?

Reply:
oh yeah, it's a typo

Reply:
Okay, I'll fix it together with the bug below yea

Reply:
fixed in https://github.com/nodefluxio/vortex/pull/43/commits/55d59038dd4cec7f86f7d85ab8b8be7a2dced7cd